vmdb/app/models/chargeback.rb

```diff
@@ -235,7 +235,7 @@ def self.get_time_range(perf, interval, tz)
 
   def self.default_rates()
     rates = []
-    ChargebackRate.find(:all, :conditions=>{:default=>true}).each do |rate|
+    ChargebackRate.find(:all, :conditions => {:default => true}).each do |rate|
```

Comment:
yeah, this is probably the 2nd of many PRs to change to arel syntax.  The rubocop space around operator commit changed this one.